src/test/resources/pages/instructorFeedbackSubmitPageAwaiting.html

```diff
@@ -258,7 +258,7 @@
               </span>
             </div>
             <div class="col-sm-10">
-              <div class="panel panel-default panel-body mce-content-body content-editor empty" data-is-session-open="false" data-length-text-id="responseLength-1-1" data-recommended-text="0" id="responsetext-1-1" name="responsetext-1-1" spellcheck="false">
+              <div class="panel panel-default panel-body mce-content-body content-editor empty" contenteditable="true" data-is-session-closed="true" data-length-text-id="responseLength-1-1" data-recommended-text="0" id="responsetext-1-1" name="responsetext-1-1" spellcheck="false" style="position: relative;">
```

Comment:
Everything is working fine, I just forgot to run CI test on Godmode to update Html files after updating Java file :sweat_smile: